### ISSUE TITLE
Add Simple BEM visualisation

### ISSUE THREAD

**Issue by mhamilt** [ca-knowledgebase/ca-knowledgebase.github.io#45]
Something along the lines of

![image](https://user-images.githubusercontent.com/33174176/174053819-39d08164-06fc-4005-8c33-fa7577dd2ecd.png)

With some animation

**Comment by mhamilt** [comment]
Where 

- Line is boundary
- Squiggle is wave
- Red dots are "speakers"
- purple dot is pint of audition

**Comment by andcic** [comment]
@jahargreaves might have something similar?

**Comment by mhamilt** [comment]
Not a work of art, but a start

https://editor.p5js.org/mhamilt/sketches/vSvK9DmVb

**Comment by jahargreaves** [comment]
Yes I have various things like this. I suggest we do it using time-domain BEM (which I have code for). I know this sounds more complicated but it actually makes the animation far easier to understand. I can help with this. You can see some approximate versions of this without diffraction at [https://youtu.be/1GxQudllHKw?t=843](https://youtu.be/1GxQudllHKw?t=843)
